server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/dash/sources/coinmarketcap/CoinmarketcapRateSource.java

```diff
@@ -66,23 +70,20 @@ public BigDecimal getExchangeRateLast(String cryptoCurrency, String fiatCurrency
         if (!getFiatCurrencies().contains(fiatCurrency)) {
             return null;
         }
-        CMCTicker[] tickers = api.getTickers(fiatCurrency);
+        CMCTicker[] tickers = api.getTickers(cryptoCurrency, fiatCurrency);
         for (int i = 0; i < tickers.length; i++) {
             CMCTicker ticker = tickers[i];
-            if (cryptoCurrency.equalsIgnoreCase(ticker.getSymbol())) {
+            if (cryptoCurrency.equalsIgnoreCase(ticker.getName())) {
```

Comment:
Revert to the original way - cryptoCurrency parameter is the coin symbol, not the full name. This wouldn't work.